classes/teacher.rb

```diff
@@ -0,0 +1,19 @@
+require 'pry'
```

Comment:
- This line makes your app throw an error and prevents it to run. Please, can you fix this, otherwise, your app works fine.